docs/Getting_Started/How_To_Test.md

```diff
@@ -2,5 +2,8 @@
 
 0. `git clone https://github.com/paypal/NNAnalytics.git`
```

Comment:
Can you add a runner script so that the executables can be kicked off via the command line? And then update this documentation.

Reply:
Done as part of https://github.com/paypal/NNAnalytics/pull/20/commits/370accd5fdcc7186a0dbf12f8bc959230233e8bd -- Please take a look.

Reply:
You should be able to validate this now by (a) running `./gradlew -PmainClass=TestNNAnalytics execute` first and then checking (with your flavor of browser) at http://localhost:4567

And then (b) checking the MiniCluster by running `./gradlew -PmainClass=TestWithMiniCluster execute` and checking http://localhost:4567 again.

The goal is that in (b) you should eventually see the NNA dashboard update with more files and directories over time. It may take some minutes to see. I have seen within 3-5 minutes that an update happens.

Reply:
The runner commands seem to be running forever. I won't block this PR but would like to see this addressed in a follow-on PR.